artmaker/src/main/java/io/artmaker/ArtMakerViewModel.kt

```diff
@@ -63,14 +62,19 @@ internal class ArtMakerViewModel(
                 key = SELECTED_STROKE_WIDTH,
                 defaultValue = 5,
             ),
+            shouldUseStylusOnly = preferences.get(
```

Comment:
The preferences should also be refactored but that is a task for some other time.

Reply:
I agree, we should add this as an Issue

Reply:
Done...